apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/engine/cloud/server-wallets/components/try-it-out.tsx

```diff
@@ -184,9 +184,14 @@ export function TryItOut() {
 }
 
 const sdkExample = () => `\
-import { createThirdwebClient, sendTransaction, getContract, Engine } from "thirdweb";
+import {
```

Comment:
can you please get these on 1 line? reads better in the docs that way, otherwise there's a wall of imports first